sql_user_check_mysql/all_tables_columns.sql

```diff
@@ -0,0 +1,36 @@
+SELECT DISTINCT 'default'                       AS schema_name,
+                c.table_name                    AS table_name,
+                COALESCE(tt.TABLE_COMMENT, '')  AS table_description,
+                c.column_name                   AS column_name,
+                c.data_type                     AS data_type,
+                COALESCE(c.column_comment, ' ') AS column_description,
+                ordinal_position                AS column_id,
+                c.table_schema                  AS `database`,
+                FALSE                           AS is_view,
+                COALESCE(tt.table_rows, 0)      AS num_rows,
+                CASE
+                    WHEN lower(c.is_nullable) = 'yes' THEN
+                        TRUE
+                    WHEN lower(c.is_nullable) = 'no' THEN
+                        FALSE
+                    END                         AS is_nullable,
+                cc.constraint_type,
+                COALESCE(tt.data_length, 0)     AS table_size
+FROM (SELECT *
+      FROM INFORMATION_SCHEMA.COLUMNS) AS c
+         LEFT JOIN INFORMATION_SCHEMA.tables AS tt ON c.table_schema = tt.table_schema
+    AND c.table_name = tt.table_name
+         LEFT JOIN (SELECT DISTINCT tc.table_schema AS SCHEMA_NAME,
+                                    kcu.table_name,
+                                    kcu.column_name,
+                                    max(tc.constraint_type)
+                                                    AS constraint_type
+                    FROM information_schema.table_constraints AS tc
+                             JOIN information_schema.key_column_usage kcu ON tc.constraint_name = kcu.constraint_name
+                    WHERE tc.constraint_type in ('PRIMARY KEY', 'FOREIGN KEY')
+                    GROUP BY tc.table_schema,
+                             kcu.table_name,
+                             kcu.column_name) AS cc ON c.column_name = cc.column_name
+    AND c.table_schema = cc.schema_name
+    AND c.table_name = cc.table_name
+ORDER BY c.table_schema, c.table_name;
```

Comment:
ORDER BY `c.table_schema`, c.table_name;

I am checking it again, on my local system if I run this it gives
```
[HY000][3065] Expression #1 of ORDER BY clause is not in SELECT list, references column 'c.TABLE_SCHEMA' which is not in SELECT list; this is incompatible with DISTINCT
```

and so I updated it to "database" in metadog, because it is renamed in select statement. but when this query is run on server it gives error and need it back as `c.table_schema`.

checking if I can fix both, otherwise I will revert it back to `c.table_schema` in metadog.
@arpit-lb @aditya-lb